2-projects/1-todo-list/script.js

```diff
@@ -1,6 +1,39 @@
 function populateTodoList(todos) {
   let list = document.getElementById("todo-list");
+
   // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
+  list.innerHTML = "";
+
+  for (let i = 0; i < todos.length; i++) {
+    let todo = todos[i];
+    let li = document.createElement("li");
+    li.innerText = todo.task;
+    li.classList.add("list-group-item");
+
+    let badge = document.createElement("span");
+    badge.classList.add("badge", "bg-primary", "rounded-pill");
+    li.appendChild(badge);
+
+    let completeBtn = document.createElement("i");
+    completeBtn.classList.add("fa", "fa-check");
+    completeBtn.setAttribute("aria-hidden", "true");
+    completeBtn.addEventListener("click", function () {
+     li.style.textDecoration =
+     li.style.textDecoration === "line-through" ? "none" : "line-through";
```

Comment:
I think you are missing something here

Reply:
Thank you for you feedback , its one line i used assignment operator and strict equality operator.
like this  `li.style.textDecoration = li.style.textDecoration === "line-through" ? "none" : "line-through";`